### ISSUE TITLE
Selectors for days of the week selectors are not accessible

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#338]
Currently, the course dates selectors for days of the week are not accessible via screen reader. This is probably because they are not rendered as `<button>`s.

**Comment by bmathews** [comment]
This should be fixed by https://github.com/ntdb/react-weekday-picker/pull/3 since the focus and key-navigation will then work the same as the react-day-picker. If it turns out we need to take it a step further, we can either add `role="button"` to avoid breaking the styles, or switch to a `<button>` element like you suggested.

**Comment by ragesoss** [comment]
I deployed the new 1.0.3 version to staging, but Helaine reports that it is still not accessible.

**Comment by ragesoss** [comment]
There is still an accessibility issue: the state (selected or not selected) of each weekday is not indicated via screen reader.

**Comment by ragesoss** [comment]
Probably the way to solve this is with `aria-state="[true/false]"`.
